EXAMPLE/Cycle-Audio-Devices.ps1

```diff
@@ -0,0 +1,30 @@
+# This script cycles through all audio devices
+
+
+# Get a list of all enabled devices
+[System.Collections.ArrayList]$audioDevicesList = Get-AudioDevice -List
+
+# Get the default playback device as <AudioDevice>
+$defaultPlaybackDevice = Get-AudioDevice -Playback
+
+#Remove non-playback devices from audioDevicesList
+for ( $index = 0; $index -lt $audioDevicesList.count; $index++ ) {
+   if ($audioDevicesList[$index].type -ne "Playback") {
+      $audioDevicesList.Remove($audioDevicesList[$index])
```

Comment:
Changing the number of item in an array while iterating through it this way using a _for_  can end up not producing the desired result. I would instead suggest something like 
`Get-AudioDevice -List | Where-Object -Property Type -EQ "Playback"`